### PR TITLE
Resource name

### PR DESCRIPTION
**Pull request by kishan1988-hub**
Updated with resource names and formatted code with terraform fmt command

### CODE REVIEW

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
Not all resources are tagged. 
Please properly name all resources that support tagging.

Also, please add env_code variable to the name.
More info about nev_code in this video tutorial:
https://youtu.be/s5e9pQXqq-U

**Comment by kishan1988-hub** [comment]
Requested changes have been made, please review

File: main.tf [hunk 1]
```diff
@@ -1,74 +1,103 @@
...
+resource "aws_vpc" "main" {
   cidr_block       = var.vpc_cidr_block
   instance_tenancy = "default"
-}
 
-locals {
-  pub_cidr = ["10.0.1.0/24", "10.0.2.0/24"]
-  pvt_cidr = ["10.0.3.0/24", "10.0.4.0/24"]
+  tags = {
+    Name = "private-${var.env_code}"
```
Comment (vladimirmukhin) [inline main.tf]:
```suggestion
    Name = var.env_code
```

File: main.tf [hunk 2]
```diff
@@ -1,74 +1,103 @@
...
+  }
 }
 
-resource "aws_internet_gateway" "myigw" {
-  vpc_id = aws_vpc.myvpc.id
+resource "aws_internet_gateway" "main" {
+  vpc_id = aws_vpc.main.id
+
+  tags = {
+    Name = "${var.env_code}"
```
Comment (vladimirmukhin) [inline main.tf]:
```suggestion
    Name = var.env_code
```

File: main.tf [hunk 3]
```diff
@@ -1,74 +1,103 @@
...
+
+  vpc_id = aws_vpc.main.id
   route {
     cidr_block = "0.0.0.0/0"
-    gateway_id = aws_internet_gateway.myigw.id
+    gateway_id = aws_internet_gateway.main.id
+  }
+
+  tags = {
+    Name = " ${var.env_code}-public"
```
Comment (vladimirmukhin) [inline main.tf]:
```suggestion
    Name = "${var.env_code}-public"
```

File: variable.tf [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
   default = "10.0.0.0/16"
 }
+
+variable "env_code" {
+  type = string
+}
+
+variable "public_cidr" {
+
+}
```
Comment (vladimirmukhin) [inline variable.tf]:
```suggestion
variable "public_cidr" {}
```

File: variable.tf [hunk 5]
```diff
@@ -1,3 +1,15 @@
...
+
+variable "private_cidr" {
+
+}
```
Comment (vladimirmukhin) [inline variable.tf]:
```suggestion
variable "private_cidr" {}
```

File: main.tf [hunk 6]
```diff
@@ -1,74 +1,103 @@
...
+    Name = "${var.env_code}-${count.index + 1} "
+  }
 }
 
 resource "aws_route_table" "public" {
-  vpc_id = aws_vpc.myvpc.id
+  count = length(var.public_cidr)
+
+  vpc_id = aws_vpc.main.id
   route {
```
Comment (vladimirmukhin) [inline main.tf:67]:
```suggestion
  vpc_id = aws_vpc.main.id
  
  route {
```

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**